packages/@aws-cdk/aws-autoscaling/lib/warm-pool.ts

```diff
@@ -0,0 +1,101 @@
+import { Resource } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { IAutoScalingGroup } from './auto-scaling-group';
+import { CfnWarmPool } from './autoscaling.generated';
+
+/**
+ * Basic properties for a warm pool
+ */
+export interface BasicWarmPoolProps {
```

Comment:
```suggestion
export interface WarmPoolOptions {
```

Reply:
To make sure, the suggestion does not align with what this package has in terms of naming convention. Here is [an example](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-autoscaling/lib/lifecycle-hook.ts). Do we want to diverge?

Reply:
Yes, lets. Looks like the example you linked was introduced over 3 years ago, which might be before we introduced the `Options` -> `Props` paradigm.

Reply:
Thanks for confirmation. Will follow the suggestion.